src/backends/file_storage.c

```diff
@@ -42,7 +43,14 @@ int open_file_storage(struct file_storage* fstorage, size_t size, const char* fi
     }
 
     /* try to load storage file content */
-    return read_from_file(fstorage->filename, fstorage->data, fstorage->size);
+    if (!netplay_is_init())
```

Comment:
You should not modify backends like that, but instead provide another backend (netplay_storage_backend ?) and configure the core to use (in main_run()) it when in netplay mode.